backend/src/api/model/event.rs

```diff
@@ -28,7 +28,7 @@ use crate::{
         util::LazyLoad,
     },
     db::{
-        types::{Credentials, EventCaption, EventSegment, EventState, EventTrack},
+        types::{Credentials, EventCaption, EventSegment, EventState, EventTrack, SeriesState},
```

Comment:
I guess this is a good opportunity to move `SeriesState` to our new `model` top level module? But not required obviously